docs/index-custom-title-page.html

```diff
@@ -0,0 +1,222 @@
+<style>
+  * {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  .card {
+    cursor: pointer;
+    padding: 16px;
+    text-align: left;
+  }
+  
+  .card:hover {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 16px;
+    text-align: left;
+  }
+  
+  .card-description {
+    color: #000;
+  }
+
+  #guide a.no-text-decoration:hover {
+    text-decoration: none!important;
+  }
+
+  .icon {
+    width: 30px;
+    height: 30px;
+    background-position: bottom;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .section-title {
```

Comment:
You should be able to remove these overrides after switching to heading tags `<h1>, <h2>...`, etc